dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/FeatureResolverImpl.java

```diff
@@ -1121,7 +1121,7 @@ private Set<String> doResolveFeatures(Collection<String> rootFeatures, Set<Strin
         // addition conflict.  That implies that a better solution may be available.
         // As long as there are more conflicts than the number of initial root conflicts
         // and there is a different permutation to try do another pass
-        while (selectionContext.currentHasMoreThanInitialBlockedCount() && selectionContext.popPermutation()) {
+        while ((selectionContext.currentHasMoreThanInitialBlockedCount() || selectionContext.currentHasMissing()) && selectionContext.popPermutation()) {
```

Comment:
Do we have an existing error message for the case where we don't find a feature chain clean of missing features?